<launch>

  <param name="use_sim_time " value="true"/>

 <node pkg="move_base" type="move_base" respawn="false" name="move_base" output="screen">

  <rosparam file="$(find jackal_navigation)/params/costmap_common_params.yaml" command="load" ns="global_costmap" />
  <rosparam file="$(find jackal_navigation)/params/costmap_common_params.yaml" command="load" ns="local_costmap" />

  <rosparam file="$(find jackal_navigation)/params/odom_nav_params/global_costmap_params.yaml" command="load" />
  <rosparam file="$(find jackal_navigation)/params/odom_nav_params/local_costmap_params.yaml" command="load" />
  
  <rosparam file="$(find jackal_navigation)/params/teb_local_planner_params.yaml" command="load" />
  <rosparam file="$(find jackal_navigation)/params/move_base_params.yaml" command="load" />

  <param name="base_global_planner" type="string" value="navfn/NavfnROS" />
  <param name="base_local_planner" value="teb_local_planner/TebLocalPlannerROS"/> 

  <remap from="laser_scan" to="/obstacle_clearing_filtered_scan" />


     <!-- <param name="base_global_planner" type="string" value="global_planner/GlobalPlanner" />
    <param name="base_local_planner" value="base_local_planner/TrajectoryPlannerROS"/> -->
    
 
  
 </node>

<!-- <node pkg="free_space_motion_tube_ros" type="barn_motion_tube_node" respawn="true" name="barn_motion_tube_node" /> -->
</launch>
